File: tests/fixtures/typescript/broken_syntax.ts
// Deliberately broken TypeScript — used to verify parse-error counting.
// tree-sitter-typescript surfaces ERROR / MISSING nodes for unmatched
// brackets, bad type annotations, and incomplete expressions.

export function ok(): number {
    return 1;
}

export function broken(x: number {
    return x + ;
}

export class Partial
    private field: = ;

    method() {

File: tests/fixtures/python/broken_syntax.py
# Deliberately broken Python — used to verify parse-error counting.
# tree-sitter-python produces ERROR / MISSING nodes for unmatched
# brackets, bad indentation, and incomplete expressions.

def ok():
    return 1


def broken(
    x =
    return x


class Partial
    def method(self):
        pass
